browser/brave_ads/ads_service_browsertest.cc

```diff
@@ -430,7 +430,7 @@ IN_PROC_BROWSER_TEST_F(BraveAdsBrowserTest,
 
 IN_PROC_BROWSER_TEST_F(BraveAdsBrowserTest,
                        BraveAdsMigrateDefaultAdsPerHourFromVersion10) {
-  EXPECT_FALSE(
+  EXPECT_TRUE(
```

Comment:
@mherrmann was this change intentional? Thanks

Reply:
@tmancey yes! I am touching the test reporting functionality here. So I purposely introduced a failing test to see whether it still reports failures correctly. It does! I'm reverting the change.